components/dashboard/ExportTransactionsCSVModal.tsx

```diff
@@ -90,12 +107,17 @@ const ExportTransactionsCSVModal = ({
     { defaultData: 0 },
   );
 
-  const handleFieldOptionsChange = ({ value }) => {
-    setFieldOption(value);
-    if (value === FIELD_OPTIONS.DEFAULT) {
-      setFields(DEFAULT_FIELDS.reduce((obj, key) => ({ ...obj, [key]: true }), {}));
-    }
-  };
+  const fieldSetOptions: Array<{ value: string; label: string; fields?: Array<string> }> = React.useMemo(() => {
+    const customFields = accountSettings?.account?.settings?.exportedTransactionsFieldSets;
+    return [
+      ...FieldOptions,
+      ...keys(customFields).map(key => ({
```

Comment:
You're assuming that `keys(null)` return `[]` ?

Reply:
Yes. This is lodash's default behavior.